data/json/recipes/practice/fabrication.json

```
@@ -0,0 +1,25 @@
+[
+  {
+    "id": "prac_fletching",
+    "type": "practice",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_FABRICATION",
+    "name": "fletching",
+    "description": "Practice fletching by modifying, studying and testing the aerodynamics of some arrows or bolts.",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "archery", 1 ] ],
+    "practice_data": { "min_difficulty": 1, "max_difficulty": 3, "skill_limit": 4 },
+    "proficiencies": [ { "proficiency": "prof_fletching", "fail_multiplier": 1, "time_multiplier": 1 } ],
+    "time": "1 h",
+    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "CUT", "level": 1 } ],
+    "//": "Modifying so much the arrows/bolts will cause them to eventually lose their shape",
+    "components": [ 
+    [ [ "arrow_fire_hardened_fletched", 4 ], [ "bolt_crude", 4 ] ],
+    [ [ "adhesive", 1, "LIST" ], [ "filament", 20, "LIST" ] ],
+    [ [ "fletching", 1, "LIST" ] ] 
```

```suggestion
      [ [ "arrow_fire_hardened_fletched", 4 ], [ "bolt_crude", 4 ] ],
      [ [ "adhesive", 1, "LIST" ], [ "filament", 20, "LIST" ] ],
      [ [ "fletching", 1, "LIST" ] ] 
```
This is from looking at the "Details" for the JSON formatting failed test, then clicking on "Show Corrections".

---

Thanks! I thought I had resolved that before submitting the PR...